hardhat/hardhat.config.ts

```diff
@@ -90,6 +90,44 @@ task('create-user-policy', 'Creates a policy for the given user')
     });
   });
 
+task('accept-claim', 'Accepts the given claim')
+  .addParam('claimId', 'The claim ID')
+  .setAction(async (args, hre) => {
+    const accounts = await hre.ethers.getSigners();
+
+    // The index for the manager needs to be in sync with the deploy script
+    const manager = accounts[1];
+    const claimsManager = ClaimsManagerFactory.connect(process.env.REACT_APP_CLAIMS_MANAGER_ADDRESS!, manager);
```

Comment:
I know it is just short term thing, but can we we check for `REACT_APP_CLAIMS_MANAGER_ADDRESS` and error our if it doesn't exist?